ONSdigital/eq-author-app
# EAR-1473-piped-meta-not-displaying-as-deleted

### What is the context of this PR?

When piping in a metadata value into a question Title and then deleting the relevant row from the metadata table, it does not display the placeholder [Deleted metadata] in the left navigation bar. 

Previously, [deleted metadata] in the question title was being handled in the front end and not updating the cache
This Pr seeks to move this process to the resolver so that the cache is updated and therefor the navigation sidebar is reRendered 😀

### How to review

1. Create a new Question in your Questionnaire
2. add in a piped metadata in the question title
3. go to metadata page and delete that metadata
4. Both the Question title _**and**_ side navbar title for that question should ready [Deleted metadata]

also test for multiple questions with piped metadata in the title when deleting the relevant metadata

Also check the Question description is updated to include [deleted metadata] if it has metadata piped that has been deleted

UNSURE IF THIS WOULD NEED A MIGRATION  ???
